### ISSUE TITLE
Why Solution Named "Stammbaum"

### ISSUE THREAD

**Issue by jogibear9988** [ststeiger/PdfSharpCore#16]
Why is the Solution Name Stambaum.sln and not "PdfSharp" or smth. similar?

**Comment by ststeiger** [comment]
Because  I originally incorporated this as part of a genealogy program. 
(I got the data from my grandfather's hobby program, and tried to integrate it with some existing genealogy program, and realized they're all crap, including but not limited to gramps, so I tried writing my own ).
I was experimenting with how best to print out a pedigree (or Stammbaum in German), for which I thought PDF wouldn't be bad, and I was working on Linux with .NET, so I searched for .NET Core PDF libraries and there were none in a functional state. 

I had since abandoned the project, because I realized graphing a pedigree (with holes) is a lot more difficult, and the best course of action was doing it with d3js and using wkhtml2pdf to create the pdf. 

Some people on stackoverflow where then complaining about being unable to create pdf reports in .NET Core, upon which I pointed them to this repository - which is how it became 'popular'.

That's why it's called pedigree - because I hacked this together for a pedigree test application, was not originally intended to share PdfSharp.
